### PR TITLE
rewrote Pilot Class Feats

### PR DESCRIPTION
**Pull request by bleehu**
And restyled them as well
	modified:   02_Classes.txt

### CODE REVIEW

File: CharacterCreation/02_Classes.txt [hunk 1]
```diff
@@ -366,27 +367,83 @@ Skill Proficiencies:
...
+    This Class Feat requires a Character to be level 10 and to have taken 
+Getaway Driver before they can take it.
+
+Boostin' Buggies:
+    
+    You have a deep familiarity with a variety of vehicle security systems and
+ignitions. 
+
+    You gain a +16 to skill checks with Slight of Hand(Dex), Hacking(Int), or
+Jerry Rig(Dex) to start and control a vehicle that didn't have the keys in it
```
Comment (Turtlelord26) [inline CharacterCreation/02_Classes.txt]: sp: Jury-rig

**Review by Turtlelord26 — APPROVED** [review]
I have GTA-CX edition concerns.

(yes, silly)

File: CharacterCreation/02_Classes.txt [hunk 2]
```diff
@@ -366,27 +367,83 @@ Skill Proficiencies:
...
+
+    You have an intuitive understanding of the flow of traffic and the lines of
+sight between your vehicle and those you'd rather avoid. Maybe you had a 
+criminal career in the past, or maybe you just see the road differently then
+most. 
+
+    Any enemy that targets the vehicle that you are piloting, the passengers in
+the vehicle that you are piloting or you whilst you are piloting takes a -2 to
+accuracy rolls to hit as long as the vehicle has moved for at least 2 Actions
+on your last turn. Furthermore you get a +12 to stealth(Dex) rolls made to lose
```
Comment (rosebrallier) [inline CharacterCreation/02_Classes.txt]: Stealth (Dex)

**Review by rosebrallier — APPROVED** [review]
Super cool!

File: CharacterCreation/02_Classes.txt [hunk 3]
```diff
@@ -366,27 +367,83 @@ Skill Proficiencies:
...
+* small boat
+* large watercraft
+* helicopter
+* starfighter
+* warship
+
+<Vehicle> Mastery:
+
+	You are THE guru across the galaxy for <Vehicles>. Add 50 onto a pertinent 
+roll about the vehicle.
```
Comment (rosebrallier) [inline CharacterCreation/02_Classes.txt]: This breaks my DC balance: 120 Extra-Crit + 60 Vehicle Skill + 50 Vehicle Mastery = 230 > 200 so pls nerf.

File: CharacterCreation/02_Classes.txt [hunk 4]
```diff
@@ -366,27 +367,83 @@ Skill Proficiencies:
...
+* tank
+* small boat
+* large watercraft
+* helicopter
+* starfighter
+* warship
+
+<Vehicle> Mastery:
+
+	You are THE guru across the galaxy for <Vehicles>. Add 50 onto a pertinent
```
Comment (rosebrallier) [inline CharacterCreation/02_Classes.txt:445]: This breaks my DC balance: 120 Extra-Crit + 60 Vehicle Skill + 50 Vehicle Mastery = 230 > 200 so pls nerf.

**Review by rosebrallier — CHANGES_REQUESTED** [review]
Wait I have balance concerns
Keep in mind that players should literally not be able to roll over 200.

File: CharacterCreation/02_Classes.txt (hunk 4 above)
Comment (bleehu) [reply CharacterCreation/02_Classes.txt:445]: Created issue https://github.com/bleehu/Compound_X/issues/695

File: CharacterCreation/02_Classes.txt [hunk 5]
```diff
@@ -366,27 +367,83 @@ Skill Proficiencies:
...
+lives with the blunt force of your bumper.
+
+    Teammates firing from a vehicle that you are piloting get a +1 to accuracy
+rolls and negate any existing penalties from chaotic vehicle movement. If you
+attempt to make an attack by ramming, crashing, scraping or splattering, you 
+gain a +2 to hit with the attack roll and any damage to your vehicle from your
+attacks is reduced to three quarters. 
+
+    This Class Feat requires a Character to be level 10 and to have taken 
+Getaway Driver before they can take it.
```
Comment (rosebrallier) [inline CharacterCreation/02_Classes.txt:398]: nitpick: "before they can take it" isn't strictly necessary to the meaning of the sentence.

File: CharacterCreation/02_Classes.txt [hunk 6]
```diff
@@ -366,27 +367,83 @@ Skill Proficiencies:
...
+
+Boostin' Buggies:
+    
+    You have a deep familiarity with a variety of vehicle security systems and
+ignitions. 
+
+    You gain a +16 to skill checks with Slight of Hand(Dex), Hacking(Int), or
+Jury Rig(Dex) to start and control a vehicle that didn't have the keys in it
```
Comment (rosebrallier) [inline CharacterCreation/02_Classes.txt]: Put spaces b/t the skill and the attribute pls :)

**Review by rosebrallier — APPROVED** [review]
No blocking comments. I love the roguish Han Solo aesthetic, and the increased variety!

File: CharacterCreation/02_Classes.txt [hunk 7]
```diff
@@ -403,7 +403,7 @@ Boostin' Buggies:
 ignitions. 
 
     You gain a +16 to skill checks with Slight of Hand(Dex), Hacking(Int), or
```
Comment (rosebrallier) [inline CharacterCreation/02_Classes.txt:405]: Moar spaces pls

**Pull request merged**